### DISCUSSION TITLE
Bounty on my head, I am hunted - non "lay low" exit route?

### DISCUSSION THREAD

**Discussion by Joachim-Otahal** [endless-sky/endless-sky#10734] [General]
I got a bounty on my head. So those bounty hunters come after me on a quite nice schedule.
They are not a problem, they are a "source of income" for me due to a weapon choice which leaves most ships intact to loot, so each attack gives me 5+ million. I could escape the first time, add three Hai Shield Beetles (I luckily had the money) to my five behemoths and three bulk freighters, and I am now at 13 Hai Shield Beetles, 11 Behemoths and five Bulk Freighters.
But is there another trigger to stop them attacking me besides "laying low"? Like "kill the one who set the bounty" or something like that? 'cause I will definitely NOT lay low and rather keep looting and killing them, but it gets repetitive.
So if I missed simple clue, would be nice to know!
(can be considered as a beginner-follow-up-question on https://github.com/endless-sky/endless-sky/issues/10720 :D )

**Comment by TheGiraffe3** [comment]
Try parking/selling some of your freight ships.
You get a price on your head when you're deemed powerful and rich enough to be worth the bother.

So if you park your "richness" (e.g. the cargo ships), the price should disappear.

**Comment by danaris** [comment]
Isn't it the opposite? Looking at the "Hunted" mission, I believe the way to "lay low" is actually to park all your _warships_.

Pirates want your cargo; bounty hunters want to take down the big scary warship.

**Comment by Joachim-Otahal** [comment]
Ah, OK :D. Where can I see whether the trigger is gone? Is it one of those four lines in the save or something else?
```
"reputation with"
	Bounty -2879.7
	"Bounty (Disguised)" 1
	"Bounty Hunter" -8582.7
	"Bounty Hunter that Won't Enter Hai Space" -1000

```
Oh, and yes, they attacked me once in Hai Space too after 20+ attacks in normal space :D.

**Reply by TheGiraffe3** [reply]
It's got to do with your `armament deterrence`, so you should be able to see it in-game.

**Comment by Joachim-Otahal** [comment]
Since I...
<details>
  <summary>Spoiler warning</summary>
  ...dominated the last pirates in New Tortuga and looted several jump drives from the Korath (Exiles?) attacking there, no bounty attack has happened again. So there is a second way :D, I just don't know whether "dominate last pirates" or "my fleet can handle Korath and loot them instead of just killing".
Right now with 17 jump drives, and other fine stuff looted from them, on that many ships going 'round the coalition area to the south west. To my surprise I did not get instantly attacked. Suppose going east of New Tortuga would be different.
</details>
But the sun is about to win against the clouds, and Germans have a habit to digest sun in winter - when it appears :D.

**Reply by TheGiraffe3** [reply]
My guess is that landing on a Republic/Syndicate/Free Worlds planet will get your hunters on your tail again.
Glad you managed to evade them though!

**Reply by Joachim-Otahal** [reply]
Oh, I did. My greed gene took over to get from 250 million pocket money to over 300 million. I saw no reason for more ships since they already dominated, so lets get even richer :D. So the usual "round trips with ultra-wealthy" sent me a few times around the spiral, and they did not reappear (yet?).
